requirements-dev.txt

```diff
@@ -2,3 +2,5 @@ black==22.3.0
 moto==4.1.7
 mypy==1.1.1
 types-mock==4.0.12
+typeguard>=4
+mypy-boto3-s3~=1.34.0
```

Comment:
I updated requirements.txt so that `awsudo` would run happily in my SGTM virtual env